### DISCUSSION TITLE
1.x 当自定义组件执行了 mutators.change 之后，外部执行setValue 貌似不起作用了

### DISCUSSION THREAD

**Discussion by Zhou-Bill** [alibaba/formily#2233] [Q&A 1.x] [answered]
[重现地址](https://codesandbox.io/s/angry-mayer-hr58t?file=/test.js:152-168)

* 先点 `外部修改value` 按钮
* 再点 `修改自定义组件value` 按钮
* 最后再点 `外部修改value` 按钮, 这一步有问题，外部value 不会重新设置

**Comment by janryWang** [comment, marked as answer]
这是因为props没变化，你加个随机串就行了
